independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/maven/workspace/LocalProject.java

```diff
@@ -249,6 +253,11 @@ public Path getSourcesSourcesDir() {
         return resolveRelativeToBaseDir(srcDir, "src/main/java");
     }
 
+    public Path getSourcesDir() {
```

Comment:
This looks like a duplicate of getSourcesSourcesDir()?

Reply:
`getSorucesSourcesDir` returns the path to `src/main/java` and this thing returns the path to `src/main`